src/Formio.js

```diff
@@ -157,6 +157,22 @@ module.exports = React.createClass({
       }
     }
   },
+  handleConditionalHideNShow: function(elementConditionalValue) {
+  if (elementConditionalValue) {
+      return true;
+    } else {
+      return false;
+   }
+  },
+  handleConditional: function (e) {
+  if (document.getElementById(e.props.component.conditional.when)) {
```

Comment:
We should probably use this.props.data[key] instead of getElementById. We are able to access the full data directly without using the dom.